lib/account/widgets/review_detail.dart

```diff
@@ -15,20 +15,22 @@ class ReviewDetail extends StatelessWidget {
   Widget build(BuildContext context) {
     final Widget header = Row(
       children: <Widget>[
-        ProfileChip(review.writer!, withHero: withHero),
-        Text(
-          localeManager.formatDate(review.createdAt!),
-          style: TextStyle(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
-        ),
         Expanded(
-          child: Align(
-            alignment: Alignment.centerRight,
-            child: CustomRatingBarIndicator(
-              rating: review.rating.toDouble(),
-              size: CustomRatingBarSize.medium,
-            ),
+          child: Row(
```

Comment:
is a Row in a row not the same as having the elements of the inner row as normal ones of the outer?